admin/includes/tinymce.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * TinyMCE Editor
+ *
+ * Custom config can be set up in /editors/tinymce/config.js
+ *
+ * @copyright Copyright 2003-2025 Zen Cart Development Team
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: DrByte New in v2.2.0 $
+ *
+ * Ref: https://www.tiny.cloud/docs/tinymce/latest/bootstrap-zip/
+ * Ref: https://www.jsdelivr.com/package/npm/tinymce
+ * Ref: https://github.com/tinymce/tinymce-dist/tags
+ * Ref: https://www.tiny.cloud/docs/tinymce/latest/upgrading/#upgrading-tinymce-self-hosted-manually
+ * Ref: https://www.tiny.cloud/docs/tinymce/latest/ui-localization/
+ */
+
+/* Editor GPL version to use (irrelevant if a non-GPL API Key is provided)
+ * This implementation activates the TinyMCE using the free JSDelivr CDN, which only works in 'GPL' mode.
+ * To use a newer version on CDN, set the number in TINYMCE_GPL_VERSION below.
+ * See https://github.com/tinymce/tinymce-dist/tags for latest
+ * but make sure it exists on https://www.jsdelivr.com/package/npm/tinymce .
+ * Sometimes not all plugins are updated on jsdelivr, so might need to fallback to prior release to avoid console errors.
+ */
+const TINYMCE_GPL_VERSION = '7.6.1';
+
+if (!defined('IS_ADMIN_FLAG')) {
+    die('Illegal Access');
+}
+
+$editor_doc_base_url = (function_exists('zen_catalog_base_link') ? zen_catalog_base_link() : '/');
+$editor_assets_dir = $editor_doc_base_url . DIR_WS_EDITORS . 'tinymce/';
+?>
+
+<!-- bof: TinyMCE Editor init -->
+<script title="TinyMCE-JQueryPlugin" src="https://cdn.jsdelivr.net/npm/@tinymce/tinymce-jquery@2/dist/tinymce-jquery.min.js"></script>
+<?php
+// Ensure API Key configuration entry is set; Can be overridden via an extra_configures or extra_datafiles file.
+if (!defined('TINYMCE_EDITOR_API_KEY')) {
+    define('TINYMCE_EDITOR_API_KEY', 'GPL');
+}
```

Comment:
You could use `zen_define_default('TINYMCE_EDITOR_API_KEY', 'GPL');`.

Reply:
There's going to be more code in this condition segment.